andela/travel_tool_front
# #160710722 Enable travel admin view all created guest house centres

#### What does this PR do?
Creates the interface for a travel admin to view accommodation centres and connects it to the back-end to fetch data.

#### Description of Task to be completed?
- Add CentreCard and CentreGrid components
- Add Accommodation view
- Style CentreCard, CentreGrid and Accommodation components
- Create reducers, sagas and action for fetching centres
- Write tests for components, reducers, actions and sagas
- Install country-data package to get country code for displaying country flags

#### How should this be manually tested?
- Clone the repository and change directory into the folder `cd travel_tool_front`
- Checkout from develop to this branch and pull the branch
- Start the application using `make start`
- Login to the application and click on the accommodation tab in the left side bar
- You should see all accommodation centres if there are any

#### Any background context you want to provide?
Ensure that your role is 'travel administrator' by updating your roleId to 29187 in the database

#### What are the relevant pivotal tracker stories?
[#160710722](https://www.pivotaltracker.com/story/show/160710722)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-27 at 8 05 36 am" src="https://user-images.githubusercontent.com/25195857/46128676-30549800-c22c-11e8-9f19-26d087ece5c8.png">

#### Questions:
N/A
